pkgs/by-name/ha/handheld-daemon/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  config,
+  fetchFromGitHub,
+  hidapi,
+  lib,
+  python3,
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "handheld-daemon";
+  version = "0.2.6";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "hhd-dev";
+    repo = "hhd";
+    rev = "5e86998fa5f558f8e14634ef62503149afcbab18";
+    hash = "sha256-+tyiXOvZXLbomhgFRKUNKGbkkkOxQKdk/kjeWZ4pvO0=";
+  };
+
+  pythonPath = with python3.pkgs; [
+    evdev
+    pyyaml
+    rich
+  ];
+
+  nativeBuildInputs = with python3.pkgs; [
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
+    hidapi
+  ];
+
+  # handheld-daemon contains a fork of the python module `hid`, so this hook
+  # is borrowed from the `hid` derivation.
+  postPatch = ''
+    hidapi=${hidapi}/lib/
+    test -d $hidapi || { echo "ERROR: $hidapi doesn't exist, please update/fix this build expression."; exit 1; }
+    sed -i -e "s|libhidapi|$hidapi/libhidapi|" src/hhd/controller/lib/hid.py
+  '';
+
+  postInstall = ''
+    mkdir -p $out/lib/udev/
+    cp -r $src/usr/lib/udev/rules.d $out/lib/udev/rules.d
```

Comment:
Make sure the output path does not need /usr, because on arch and fedora it does.

Reply:
Not all udev rules are meant to be copied for install. 

Only the non-user variant should if `hhd` runs as a system service, otherwise you introduce a tiny vulnerability and expand the scope of permissions.